code/js/controllers/SpotifyController.js

```diff
@@ -1,33 +1,19 @@
 ;(function() {
   "use strict";
 
-  var BaseController = require("BaseController"),
-      _ = require("lodash");
+  var BaseController = require("BaseController");
 
-  var multiSelectors = {
-    play: [null, ".now-playing-bar button[class*=play]"],
-    pause: [null, ".now-playing-bar button[class*=pause]"],
-    playPause: ["#play-pause", null],
-    playNext: ["#next", ".now-playing-bar button[class*=skip-forward]"],
-    playPrev: ["#previous", ".now-playing-bar button[class*=skip-back]"],
-    playState: ["#play-pause.playing", ".now-playing-bar button[class*=pause]"],
-    iframe: ["#app-player", null],
-    like: [".thumb.up", null],
-    dislike: [".thumb.down", null],
-    song: ["#track-name", ".now-playing-bar div div [href*='/album/']"],
-    artist: ["#track-artist", ".now-playing-bar div div [href*='/artist/']"]
-  };
-
-  var controller = new BaseController({
-    siteName: "Spotify"
+  new BaseController({
+    siteName: "Spotify",
+    play: ".control-button[title*='Play']",
```

Comment:
We can't use `title` here it needs to use the class selector. The reason is that Spotify localizes the title attribute, so it will be different for non English languages